Orange/preprocess/transformation.py

```diff
@@ -107,7 +107,10 @@ def __init__(self, variable, offset, factor):
         self.factor = factor
 
     def transform(self, c):
-        return (c - self.offset) * self.factor
+        if sp.issparse(c):
+            return c
```

Comment:
I don't understand this: why don't you normalize sparse values? In this case this transformation doesn't work as intended, does it?